### PR TITLE
fixed display top nav menu when user does not have rights for first or last module

### PR DESCRIPTION
**Pull request by tkuska**
I am targeting this branch, because  this feature is backwards compatible.

## Changelog
```markdown
### Fixed
- wrong HTML generated (top nav menu), if the user does not have rights for first or last module
```

## Subject
Fixed top menu rendering. When user does not have rights for first or last admin module, wrong HTML was generated whitch breaks top menu and user block

### CODE REVIEW

**Comment by OskarStark** [comment]
Can you provide before/after screenshots?

**Comment by tkuska** [comment]
Before
<a href="https://ibb.co/kioPpK">before</a>
After:
<a href="https://ibb.co/dkN63e">after</a> 
After expanded:
<a href="https://ibb.co/ciNjpK">after_epanded</a>

**Comment by OskarStark** [comment]
Thank you, can you please upload them to github directly?
The look very bad:
screenshot 2018-10-01 11 52 27

**Comment by tkuska** [comment]
because i blur them look upper right corner the job is done there :)

File: src/Resources/views/Core/add_block.html.twig [hunk 1]
```diff
@@ -16,27 +16,29 @@
     {% set column_count = (groups|length / items_per_column)|round(0, 'ceil') %}
 
     <div class="dropdown-menu multi-column dropdown-add"
-        {% if column_count > 1 %}style="width: {{ column_count*140 }}px;"{% endif %}
-            >
+         {% if column_count > 1 %}style="width: {{ column_count*140 }}px;"{% endif %}
```
Comment (core23) [inline src/Resources/views/Core/add_block.html.twig]:
```suggestion
        {% if column_count > 1 %}style="width: {{ column_count*140 }}px;"{% endif %}
```

File: src/Resources/views/Core/add_block.html.twig [hunk 2]
```diff
@@ -16,27 +16,29 @@
...
+                {% if render_first_element or loop.index0 % items_per_column == 0 %}
+                    {% if render_first_element %}
                         <div class="container-fluid">
                             <div class="row">
+                        {% set render_first_element = false %}
                     {% else %}
                         </ul>
-                    {% endif %}
-
+                    {%  endif %}
```
Comment (core23) [inline src/Resources/views/Core/add_block.html.twig]:
```suggestion
                    {% endif %}
```

**Comment by OskarStark** [comment]
Thank you 🙏

**Pull request merged**